examples/00-basic-examples/Maxwell3DTeam3.py

```diff
@@ -0,0 +1,188 @@
+# NB, Eddy effect setting is ignored for stranded conductor type used in Search Coil
+M3D.eddy_effects_on(["LadderPlate"], activate=True)
+M3D.eddy_effects_on(["SearchCoil"], activate=False)
+
+################################################################################
+# Add a linear parametric sweep for the two coil positions
+sweepname = "CoilSweep"
+M3D.opti_parametric.add_parametric_setup("Coil_Position", "LIN -20mm 0mm 20mm", parametricname=sweepname)
+
+################################################################################
+# For now we need to use Optimetrics module to tick 'Save Fields' box
+Opt = M3D.ooptimetrics
+Opt.EditSetup(
+    "CoilSweep",
+    [
+        "NAME:CoilSweep",
+        "IsEnabled:=",
+        True,
+        ["NAME:ProdOptiSetupDataV2", "SaveFields:=", True, "CopyMesh:=", False, "SolveWithCopiedMeshOnly:=", True],
+        ["NAME:StartingPoint"],
+        "Sim. Setups:=",
+        ["Setup1"],
+        [
+            "NAME:Sweeps",
+            [
+                "NAME:SweepDefinition",
+                "Variable:=",
+                "Coil_Position",
+                "Data:=",
+                "LIN -20mm 0mm 20mm",
+                "OffsetF1:=",
+                False,
+                "Synchronize:=",
+                0,
+            ],
+        ],
+        ["NAME:Sweep Operations"],
+        ["NAME:Goals"],
+    ],
+)
+
+# Solve the model, we solve the parametric sweep directly so results of all variations are available.
+M3D.analyze_setup(sweepname)
+
+###############################################################################
+# Postprocessing Field Plots and Graphs
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+################################################################################
+# Use fields calculator to create expression for Bz
+Fields = M3D.odesign.GetModule("FieldsReporter")
+Fields.EnterQty("B")
+Fields.CalcOp("ScalarZ")
+Fields.EnterScalar(1000)
+Fields.CalcOp("*")
+Fields.CalcOp("Smooth")
+Fields.AddNamedExpression("Bz", "Fields")
+
+###############################################################################
+# Plot mag(Bz) as a function of frequency for both coil positions
+Plot = M3D.odesign.GetModule("ReportSetup")
+Plot.CreateReport(
+    "mag(Bz) Along 'Line_AB' Offset Coil",
+    "Fields",
+    "Rectangular Plot",
+    "Setup1 : LastAdaptive",
+    ["Context:=", "Line_AB", "PointCount:=", 1001],
+    ["Distance:=", ["All"], "Freq:=", ["All"], "Phase:=", ["0deg"], "Coil_Position:=", ["-20mm"]],
+    ["X Component:=", "Distance-60mm", "Y Component:=", ["mag(Bz)"]],
+)
+
+Plot.CreateReport(
+    "mag(Bz) Along 'Line_AB' Centred Coil",
+    "Fields",
+    "Rectangular Plot",
+    "Setup1 : LastAdaptive",
+    ["Context:=", "Line_AB", "PointCount:=", 1001],
+    ["Distance:=", ["All"], "Freq:=", ["All"], "Phase:=", ["0deg"], "Coil_Position:=", ["0mm"]],
+    ["X Component:=", "Distance-60mm", "Y Component:=", ["mag(Bz)"]],
+)
```

Comment:
good this custom report. we need to wrap it better in the future in M3D.post.create_rectangular_plot method